index.html

```diff
@@ -4,9 +4,48 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles/index.css" rel="stylesheet" type="text/css"/>
     <title>Weather Report</title>
 </head>
 <body>
+    <main class="container">
+        <section id="box1">
+           <h1>Weather Report</h1>
+           <p>For the lovely city of <span id="currentCity">✨My Hometown✨</span></p>
+        </section>
+        
+        <section class="box3" class="input-boxes">
```

Comment:
Two class attributes?

```suggestion
        <section class="box3 input-boxes">
```